packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/reminderCardTermsHelpers.tsx

```diff
@@ -0,0 +1,89 @@
+import { ReactNode } from 'react';
```

Comment:
Consider importing ReactNode with `import type `

Reply:
Cool, go ahead and fix it!

Reply:
fixed

Reply:
fixed